### PR TITLE
Feat/plugin defaults

### PR DESCRIPTION
**Pull request by maimonav**

### CODE REVIEW

File: packages/plugin-image/web/src/image-component.jsx [hunk 1]
```diff
@@ -113,7 +113,9 @@ class ImageComponent extends React.Component {
   handleFilesAdded = (blockKey, { data, error }) => {
     const imageData = data.length ? data[0] : data;
     const config = { ...this.props.componentData.config };
-    config.alignment = imageData.width >= 740 ? 'center' : 'left';
+    if (!config.alignment) {
```
Comment (bchmn) [inline packages/plugin-image/web/src/image-component.jsx]: why is this change necessary? prior to your change config.alignment was always set

File: packages/plugin-image/web/src/image-viewer.jsx [hunk 2]
```diff
@@ -42,7 +42,7 @@ class ImageViewer extends React.Component {
   getImageUrl(src) {
     const { helpers, shouldRenderOptimizedImages } = this.context || {};
 
-    if (!src && helpers?.handleFileSelection) {
+    if (!src && helpers && helpers.handleFileSelection) {
```
Comment (bchmn) [inline packages/plugin-image/web/src/image-viewer.jsx]: revert

File: packages/plugin-image/web/src/image-viewer.jsx [hunk 3]
```diff
@@ -61,7 +61,7 @@ class ImageViewer extends React.Component {
         requiredHeight = this.calculateHeight(SEO_IMAGE_WIDTH, src);
       } else if (this.state.container) {
         const { width } = this.state.container.getBoundingClientRect();
-        requiredWidth = width || src.width || 1;
+        requiredWidth = width || (src && src.width) || 1;
```
Comment (bchmn) [inline packages/plugin-image/web/src/image-viewer.jsx]:
```suggestion
        requiredWidth = width || (src?.width) || 1;
```

File: packages/plugin-image/web/src/image-component.jsx (hunk 1 above)
Comment (maimonav) [reply packages/plugin-image/web/src/image-component.jsx]: Because now you may have different alignment.
